packages/ecs-init/0002-docker-do-not-bind-mount-missing-plugin-dirs.patch

```diff
@@ -0,0 +1,35 @@
+From ecfb953b6b93c6e497303a7f7d32950d5d9f28f6 Mon Sep 17 00:00:00 2001
```

Comment:
This seems like an "agility" improvement that could/should be submitted to upstream.

Do the downstream code paths handle the missing path as well? Could this cause issues for the Agent process?

Reply:
The agent runs successfully without this directory mounted in.  This may be a candidate for upstream inclusion.

Reply:
This is no longer applicable as I have moved away from using ecs-init.